.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  ![Welcome Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:8ff47a85-7250-4d86-8e48-2f346b48b2c1:BannerWelcome.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  :tada: Welcome to the 2020 OHBM Hackathon! :tada:
+  We're really excited to have your input on this repository and welcome your idea! :sparkling_heart:
+  If you haven't done so already, please make sure you check out our [README file](https://github.com/ohbm/hackathon2020/blob/master/README.md) and [Code of Conduct](https://github.com/ohbm/hackathon2020/blob/master/code_of_conduct.md).
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  ![Thank You Banner](https://zenodo.org/api/iiif/v2/0c0188d3-d03c-4830-a6e3-00405f5c22fa:7fbd97cf-283b-480c-b8e1-11866e26245c:BannerThanks.jpg/full/750,/0/default.jpg)
+  (Image: CC-BY license, The Turing Way Community, & Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332808)
+  
+  :sparkling_heart: Thanks for opening this pull request! :sparkling_heart:
+  We really appreciates your time and effort to contribute to the project.
+  Please make sure you have read our [https://github.com/ohbm/hackathon2020/blob/master/README.md) and shared details in your pull request.
```

Comment:
Link is broken: "Please make sure you have read our [Readme] (link) ..."
What kind of details do you mean here? "...and shared all relevant details in your pull request."?